scss/modules/_header.scss

```diff
@@ -8,599 +8,186 @@
           width: 50%;
           margin: 0;
-          float: left;
-          padding-left: 30px;
-
-          a,
-          a:link,
-          a:visited {
-            box-sizing: border-box;
-            padding: 10px 10px 10px 0;
-            display: block;
-            width: 100%;
-            height: 100%;
-          }
-
-          &.active {
-            a,
-            a:link,
-            a:visited {
-              color: $warm-grey;
-              font-weight: 700;
+          border-right: 1px solid lighten($mobile-nav, 60%);
+          border-bottom: 1px solid lighten($mobile-nav, 60%);
+          background: transparent;
+
+          @media only screen and (min-width: $breakpoint-large) {
+            width: auto;
+            border-right: 0;
+            border-bottom: 0;
+            border-left: 1px solid darken($brand-color, 8%);
+
+            &:last-child {
+              border-right: 1px solid darken($brand-color, 8%);
+
+              a {
+                border-right: none;
+              }
             }
           }
 
-          &.single-link {
-            width: 100%;
-          }
-        } /* end li */
-
-        li:only-child {
-          width: 100%;
-        }
-
-      } /* end third-level-nav */
-
-      .breadcrumb {
-        box-sizing: border-box;
-        width: 100%;
-        margin-left: 0;
-        margin-bottom: 0;
-
-        li:first-of-type {
-          border-bottom: 1px solid lighten($brand-color, 7%);
-          margin-bottom: -1px;
-        }
-
-        li {
-          box-sizing: border-box;
-          color: $white;
-          display: block;
-          height: 40px;
-          margin: 0;
-          width: 100%;
-
-          + li {
-            display: none;
-
-            a:link,
-            a:active,
-            a:visited {
-              padding-left: 20px;
-            }
-
-            .after {
-              background-image: none;
+          @media only screen and (max-width: $breakpoint-large) {
+            &:nth-child(2n+2) {
+              border-right: 0;
             }
           }
 
           a,
           a:link,
           a:visited {
-            box-sizing: border-box;
-            color: $warm-grey;
+            font-size: 1em;
+            font-weight: 300;
+            position: relative;
             display: block;
-            font-size: 16px;
-            margin-right: 0;
-            padding: 8px 10px 0;
+            margin-bottom: 0;
+            padding: 8px 10px;
+            text-align: left;
             text-decoration: none;
-            width: 100%;
-          }
-
-          &.active {
-            margin-top: 12px;
+            background-color: lighten($mobile-nav, 72%);
+            font-smoothing: subpixel-antialiased;
 
-            a,
-            a:link,
-            a:visited {
-              color: $warm-grey;
-              font-weight: 700;
+            @if $header-link-color != '#333' {
+              color: $cool-grey
```

Comment:
This style does not seem to be set even it the header link coor is not cool-grey? See pic in review.